specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -4851,6 +4856,24 @@
       ],
       "type": "object"
     },
+    "ImageMetadata": {
+      "type": "object",
+      "description": "Specifies the Image Metadata about current ComputeInstance",
+      "properties": {
+        "currentImageVersion": {
+          "type": "string",
+          "description": "Indicates the current version of OS Base Image, Compute Instance is running on."
+        },
+        "latestImageVersion": {
+          "type": "string",
+          "description": "Indicates the latest OS Base Image available for the Compute Instance."
+        },
+        "isLatestOsImageVersion": {
```

Comment:
Let's drop `Os` for consistency with the other properties

Reply:
Won't be possible as contract already changed in backend layer.

Reply:
typically, backend implementation shouldn't be a reason to worsen our customer-facing contract definitions

Reply:
ok... is this just basically displaying `currentImageVersion==latestImageVersion`...?

Reply:
That is correct. One of the goals it that we will be able to track compliance of compute instance states using Azure Policy. Azure Policies can be defined over boolean properties, but comparing and looking up two values for current==latest seems not an option syntactically.